modules/OFConnectionManager2/module/src/ofconnectionmanager.c

```diff
@@ -229,6 +229,35 @@ ind_cxn_send_cxn_list(void)
 
 
 /*
+ * To help debug controller-switch connection issues,
+ * syslog a space-separated list of connected controllers.
+ */
+void
+ind_cxn_syslog_active_controllers(void)
+{
+    int id;
+    controller_t *controller;
+    /* +1 for space between controller descriptions */
+    char buf[MAX_CONTROLLERS*(MAX_CONTROLLER_DESC_LEN+1)] = { 0 };
```

Comment:
the snprintf from proto_ip_string is already enforcing a null byte onto the end of each controller desc.